onnx_halide/accelerated_generator.py

```diff
@@ -0,0 +1,94 @@
+from .halide_generator import HalideGraphVisitor, HalideNodeVisitor
+from .types import VI
+import numpy as np
+import os
+from os.path import join, dirname
+from . import __path__
+from math import floor, ceil
+from .environment_link import Environment
+from .base_generator import BaseNodeVisitor
+
+from onnx.onnx_ml_pb2 import NodeProto, TypeProto
+from typing import Dict, List, Set, Tuple, Union
+class AcceleratedGraphVisitor(HalideGraphVisitor):
+    '''Manually make a copy since node_lookup is a class var'''
```

Comment:
This seems like a issue with the way `node_lookup` is defined. This is fine for now, but is there a way to define `node_lookup` and the `register` method to avoid this manual inheritance?

Reply:
`node_lookup` is defined as static in `BaseGraphVisitor`, so any changes made to it are going to be shared between `HalideGraphVisitor` and `AcceleratedGraphVisitor`. An alternate way other than manually copying it in each subclass would be to make it as a member instead and have all the graph visitors be some sort of singleton but that seems like overkill.